R/lockfile.R

```diff
@@ -243,5 +243,19 @@ renv_lockfile_records <- function(lockfile) {
   invisible(x)
 }
 
+renv_lockfile_resolve <- function(lockfile) {
```

Comment:
Not used anywhere?

Reply:
It does get used here:

https://github.com/rstudio/renv/blob/02a54315df2408efb1ad7216e8a3438eb4ab0e58/R/repair.R#L51
https://github.com/rstudio/renv/blob/02a54315df2408efb1ad7216e8a3438eb4ab0e58/R/restore.R#L59

but in each case that's after it's already been resolved so we can probably just remove those.